accelerator/config.c

```diff
@@ -477,5 +493,29 @@ void ta_core_destroy(ta_core_t* const core) {
   logger_destroy_client_core();
   logger_destroy_client_extended();
   logger_destroy_json_serializer();
-  br_logger_release();
+}
+
+void ta_logger_switch(bool quiet, bool init) {
```

Comment:
I don't think it is a matter of "quiet". Instead, it looks like "disabling" for logging facilities.

Reply:
I'll rename quiet to disable